cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -43,44 +43,138 @@ def verify_vm_power_state(vm, state, minutes=10, action=None):
         message='Wait for VM to {} (current state: {})'.format(state, vm.power_state))
 
 
+def get_collection(vm, rest_api):
```

Comment:
I would like to see this function/method in the rest_api module so it can be reused in other places